pkg/sinks/blackhole/blackhole.go

```diff
@@ -80,6 +83,18 @@ func NewBlackhole(vertex *dfv1.Vertex, fromBuffer isb.BufferReader, fetchWaterma
 	return bh, nil
 }
 
+func (b *Blackhole) getSinkGoWhereDecider() forward.GoWhere {
```

Comment:
Each sink needs to implement something like this?

Reply:
done